docs/source/notes/broadcasting.rst

```diff
@@ -11,22 +11,19 @@ automatically expanded to be of equal sizes (without making copies of the data).
 
 General semantics
 -----------------
-Two tensors are "broadcastable" if the following rules hold:
-
-- Each tensor has at least one dimension.
-- When iterating over the dimension sizes, starting at the trailing dimension,
-  the dimension sizes must either be equal, one of them is 1, or one of them
-  does not exist.
+Two tensors are "broadcastable" if when iterating over the dimension sizes,
+starting at the trailing dimension, the dimension sizes must either be equal,
+one of them is 1, or one of them does not exist.
 
 For Example::
 
     >>> x=torch.empty(5,7,3)
     >>> y=torch.empty(5,7,3)
     # same shapes are always broadcastable (i.e. the above rules always hold)
 
-    >>> x=torch.empty((0,))
+    >>> x=torch.empty(())
```

Comment:
This example looks good to show that the original first rule isn't actually true.

Now that the two examples are both positives, it might be useful to add a third negative example, just for clarity? (e.g. a case where two tensors are not broadcastable)